### ISSUE TITLE
Drop support for configuration states

### ISSUE THREAD

**Issue by rosberry** [rosberry/Framezilla#2]
Guys, what do you think about getting rid of `view.nx_state` and everything else related to it? 
It doesn't seem to be the best way to do anything and looks more like a hack.
Is anybody even using it?

**Comment by artemnovichkov** [comment]
I agree because I use it rarely. And it's a good point that we remove state check for simple `configureFrame`
